resources/assets/js/components/Application/Experience/Experience.tsx

```diff
@@ -0,0 +1,742 @@
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/camelcase */
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import {
+  Skill,
+  ExperienceEducation,
+  ExperienceWork,
+  ExperienceCommunity,
+  Experience,
+  ExperiencePersonal,
+  ExperienceAward,
+  ExperienceSkill,
+} from "../../../models/types";
+import { localizeFieldNonNull, getLocale } from "../../../helpers/localize";
+import { SkillTypeId } from "../../../models/lookupConstants";
+import EducationExperienceModal, {
+  messages as educationMessages,
+  EducationType,
+  EducationStatus,
+  EducationExperienceSubmitData,
+} from "../ExperienceModals/EducationExperienceModal";
+
+import { EducationSubformProps } from "../ExperienceModals/EducationSubform";
+import WorkExperienceModal, {
+  messages as workMessages,
+  WorkExperienceSubmitData,
+} from "../ExperienceModals/WorkExperienceModal";
+import CommunityExperienceModal, {
+  messages as communityMessages,
+  CommunityExperienceSubmitData,
+} from "../ExperienceModals/CommunityExperienceModal";
+import PersonalExperienceModal, {
+  messages as personalMessages,
+  PersonalExperienceSubmitData,
+} from "../ExperienceModals/PersonalExperienceModal";
+import AwardExperienceModal, {
+  messages as awardMessages,
+  AwardRecipientType,
+  AwardRecognitionType,
+  AwardExperienceSubmitData,
+} from "../ExperienceModals/AwardExperienceModal";
+import ExperienceEducationAccordion from "../ExperienceAccordions/ExperienceEducationAccordion";
+import ExperienceWorkAccordion from "../ExperienceAccordions/ExperienceWorkAccordion";
+import ExperienceCommunityAccordion from "../ExperienceAccordions/ExperienceCommunityAccordion";
+import ExperiencePersonalAccordion from "../ExperienceAccordions/ExperiencePersonalAccordion";
+import ExperienceAwardAccordion from "../ExperienceAccordions/ExperienceAwardAccordion";
+import { mapToObject, hasKey } from "../../../helpers/queries";
+import { ExperienceSkill as RelevantSkill } from "../ExperienceAccordions/BaseExperienceAccordion";
+
+// TODO: Move method to Experience selectors file or utility file.
```

Comment:
I can migrate these into the helpers file I created in the Review PR if you'd like

Reply:
Ya sure, thank you 👍